source/sync/configure/enable-sync.txt

```diff
@@ -219,5 +213,21 @@ Procedure
                curl https://services.cloud.mongodb.com/api/admin/v3.0/groups/{GROUP_ID}/apps/{APP_ID}/services/{MongoDB_Service_ID}/config \
                  -X PATCH \
                  -h 'Authorization: Bearer <Valid Access Token>' \
-                 -h "Content-Type: application/json"
-                 -d @/sync/config.json
+                 -h "Content-Type: application/json" \
+                 -d '<Flexible Sync Configuration>'
```

Comment:
```suggestion
                 -d '<Updated Flexible Sync Configuration>'
```
?

Reply:
Instead, changing the name for section to "Add a sync configuration", as this is for first enabling sync.